baklava/package-lock.json

```diff
@@ -0,0 +1,5966 @@
+{
```

Comment:
You are re-introducing `package-lock.json` which isn't a good idea as we have switched over to yarn.

Reply:
this is just a side note. the big comment I've written above is still the decision.